include/dpp/utility.h

```diff
@@ -275,14 +275,36 @@ namespace dpp {
 
 		/**
 		 * @brief Convert ints to RGB for sending in embeds
-		 * 
+		 *
 		 * @param red red value, between 0 and 255 inclusive
 		 * @param green green value, between 0 and 255 inclusive
 		 * @param blue blue value, between 0 and 255 inclusive
 		 * @return uint32_t returned integer colour value
 		 */
 		uint32_t DPP_EXPORT rgb(int red, int green, int blue);
 
+	        /**
+		 * @brief Convert doubles to CMYK for sending in embeds
+		 *
+		 * @param c cyan value, between 0 and 1 inclusive
+		 * @param m magenta value, between 0 and 1 inclusive
+		 * @param y yellow value, between 0 and 1 inclusive
+		 * @param k key (black) value, between 0 and 1 inclusive
+		 * @return uint32_t returned integer colour value
+		 */
+		uint32_t DPP_EXPORT cmyk(double c, double m, double y,double k);
+		
+		/**
+		 * @brief Convert ints to CMYK for sending in embeds
+		 *
+		 * @param c cyan value, between 0 and 255 inclusive
+		 * @param m magenta value, between 0 and 255 inclusive
+		 * @param y yellow value, between 0 and 255 inclusive
+		 * @param k key (black) value, between 0 and 255 inclusive
+		 * @return uint32_t returned integer colour value
+		 */
+		uint32_t DPP_EXPORT cmyk(int c, int m, int y,int k);
```

Comment:
`int` has a 4-byte size typically and a much larger range than 0-255. Choose another type.

Reply:
Brain sad to do it like that, so i did

Reply:
Probably i can ask for char, so okay, i will change it

Reply:
Why `char`? Use a n-bit `int` type.

Reply:
Also, if thinking that way i will also change rgb function. Because it has just int and getting values from 0-255.

Reply:
here is same problem as with rgb, it do not work if it not int, so let it be it.

Reply:
![image](https://user-images.githubusercontent.com/79337999/207559443-87a6a2e9-4f2e-4095-80e4-b5106e89b0e2.png)

i will try to explain my opinion. with uint8_t (which is 1 byte int to store 0-255) you cant do simple calls like on picture. Moreover we already have rgb function which is using int for same range of numbers in input. Also In my opinion having one function with int another with uint8t will look strange.

Reply:
this is why we use int, because it will confuse the user when they try and use constant values. We dont want them to have to explicitly cast every time, or put `.0` on the end of each number...